### DISCUSSION TITLE
Bearer Token

### DISCUSSION THREAD

**Discussion by skydashnet** [masterking32/MasterHamsterKombatBot#3] [Q&A] [answered]
How to get Bearer Token sir? thanks you

**Comment by masterking32** [comment, marked as answer]
> How to get Bearer Token sir? thanks you

Hello, Please [watch this video](https://www.youtube.com/watch?v=cjWE7DmMFgw), Although the language of the video is Persian, the visual content should be understandable.

**Reply by skydashnet** [reply]
There is one more problem that has arisen, sir. that is, when the script has been running for several hours... the script is stuck at 'getting account config data'.
and when I stop the script it will be normal again. is this normal?

**Reply by masterking32** [reply]
> There is one more problem that has arisen, sir. that is, when the script has been running for several hours... the script is stuck at 'getting account config data'. and when I stop the script it will be normal again. is this normal?

Use the latest version. That's not normal; I've been using that project on a server for a few days, and it's working fine for me.
